static/admin/css/autocomplete.css

```diff
@@ -0,0 +1,275 @@
+select.admin-autocomplete {
```

Comment:
static folder should be in gitignore file. Also please remove this folder from git